paasta_tools/cli/schemas/kubernetes_schema.json

```diff
@@ -285,7 +286,8 @@
                                                 "uwsgi",
                                                 "cpu",
                                                 "piscina",
-                                                "if metrics_provider is arbitrary_promql, the prometheus_adapter_config parameter is required"
```

Comment:
I think this probably needs to go at the end, from the comment above this? Not entirely sure on that though, just a guess

Reply:
Order shouldn't matter. `enum` is a set of fixed values used in validation.

http://json-schema.org/understanding-json-schema/reference/generic.html#enumerated-values
>  The enum keyword is used to restrict a value to a fixed set of values

But leaving it at the end makes more sense visually.

Reply:
Ok, fair enough, I don't really know how this string helps with the error message anyways